### ISSUE TITLE
Issues with setting ControlNet components using ConfigPresets (see issue on CN repo)

### ISSUE THREAD

**Issue by marcsyp** [Zyin055/Config-Presets#44]
Hi all,

Don't know if any of you have encountered this, but ConfigPresets and ControlNet do not play 100% nice together.  After setting the ControlNet components with ConfigPresets, everything works perfectly, but only with a workaround (Check and recheck the Pixel Perfect checkbox) due to some of the change listener behavior on ControlNet's components.  I recently logged an issue in the ControlNet repo to see if it was something they could fix, and they said no, not for now at least. 

Curious to hear your thoughts whether there's anything we could do on the ConfigPresets side, maybe we could collaborate to support ControlNet specifically, since it is one of A1111's most powerful features.

Let me know what you think, here is the issue with all the details:
https://github.com/Mikubill/sd-webui-controlnet/issues/1297

**Comment by Zyin055** [comment]
I had noticed that when Automatic1111 bumped the Gradio version to 3.23 that the script dropdown .change() event stopped working when controlling the `script_list` component. The selected script does not expand it's UI after changing the dropdown.

This is likely the same issue with ControlNet .change() events, and I am not sure how to resolve it. It's likely an internal bug/issue with Gradio, so maybe a future update will solve the problem.

**Comment by marcsyp** [comment]
Actually it's interesting because my script_list tracking works fine but the element setting on CN doesn't. But no worries, I can live with the workaround, it's still a huge time saver to use Config Presets with control net.  Thanks!

**Comment by jmilez** [comment]
Just installed this last night to find it has an issue... LOL.  The txt2img seems to work for me, but img2img is more involved.  I loaded my preset and noticed it wasn't working because it only took 2 seconds to scale the image instead of 10.  I use the img2img CN resample/tile method with Resize By scale set to 2.  

I saved the resize mode (Resize by) and scale to the preset, but when the preset is loaded, it doesn't change the gui to the Resize By method (but the 2x scale is there).  Just have to click the Resize By, but then had to basically reselect the resample and tile models from the CN menus for it to work. (toggling pixel perfect refreshed the menus, but still didn't seem to work for me)  So probably related to the gradio control update issue.
![image](https://github.com/Zyin055/Config-Presets/assets/2197703/72b2606b-b8ed-4a72-869b-cdddf3bc3de0)

Really needed this preset because Auto1111 has to be restarted frequently when upscaling images with this method because the status indicators start to count to infinity when sending images between the tabs and becomes unusable until web page is reloaded.
